src/rhsm/connection.py

```diff
@@ -572,6 +575,25 @@ def __init__(
         elif token:
             self.headers["Authorization"] = "Bearer " + token
 
+    def close_connection(self):
+        """
+        Try to close connection to server
+        :return: None
+        """
+        if self.__conn is not None:
+            log.debug(f"Closing HTTPs connection {self.__conn}")
```

Comment:
Small thought, any reason the "s" is lowercase in HTTPS here?

Reply:
Fixed